judgments/templatetags/search_results_filters.py

```diff
@@ -33,7 +18,7 @@ def is_exact_title_match(result, query):
 
 @register.filter
 def is_exact_ncn_match(result, query):
-    return preprocess_ncn(query) == preprocess_ncn(result.neutral_citation)
+    return _is_exact_ncn_match(result, query)
```

Comment:
I'm not sure why I'm not liking this name. I think I'd prefer

```
import judgments.utils
return judgments.utils.is_exact_ncn_match
```

Reply:
> In Python, single underscore `_` before an object name means that the object is meant to be private, and shouldn't be directly accessed from outside the class.

is I think why I don't like it; to me it's implying it's a private variable, rather than a name-mangled public externa one.